scripts/flattener.py

```diff
@@ -356,10 +357,11 @@ def gather_pooled_metadata(obj_type, properties, values_to_add, objs):
 							else:
 								sys.exit("Error in getting development_slims as development_stage ontology: {}".format(query_url))
 						#values_to_add[key] = dev_in_all[0]
+				
 				else:
 					sys.exit("Cxg field is a list")
 			else:		
-				values_to_add[key] = 'pooled samples: [{}]'.format(','.join(value_str))
+				values_to_add[key] = 'pooled: [{}]'.format(','.join(value_str))
```

Comment:
Pending what Jason says on Slack, but what I remember is that we said the format would be "pooled['value1','value2']". So, it should be: 
values_to_add[key] = 'pooled[{}]'.format(','.join(value_str))